### DISCUSSION TITLE
Student Developer Pack approved but no benefit.

### DISCUSSION THREAD

**Discussion by RealYusufIsmail** [community/community#67884] [GitHub Education] [answered]
### Hi everyone,

Hello. I got approved for GitHub education about 1 and a half days ago but received no benefits.
<img width="450" alt="Screenshot 2023-09-21 at 20 49 56" src="https://github.com/community/community/assets/67903886/8cf6fba3-7d40-4d82-96e1-031e75b3f548">

Also its quite strange that I received no email conformation which is quite different to the last time I applied which was about 2 years ago.

**Comment by mantasu** [comment, marked as answer]
Hey, I had the same issue, you just have to wait for a few more days as there will be some time before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by RealYusufIsmail** [reply]
Yea, thank you for the advice. Got my perks yesterday.

**Comment by dev348** [comment]
HI, I got approved for GitHub education on 23 Sep but received no benefits.

![image](https://github.com/community/community/assets/47909485/10a97796-217d-4ca5-860a-454b9496c78b)

Also it's quite strange that I received no email confirmation which is quite different to the last time I applied which was about 3 years ago.

**Reply by RealYusufIsmail** [reply]
You have to wait 3 days
